C++/API/API.cpp

```diff
@@ -1158,30 +1324,30 @@ bool iSENSE::append_email_byName(string dataset_name)
 // It formats the upload string
 // Users should not have to call this function - API methods will,
 // and will pass an int value indicating which API method they are using.
-void iSENSE::format_upload_string(int key)
+void iSENSE::format_upload_string(int post_type)
 {
   // Add the title + the correct formatting
   upload_data["title"] = value(title);
 
-  // This is now a switch. Any future API methods can be added here.
-  switch(key)
+  // This is now a switch. Future API methods can be added here.
+  switch(post_type)
   {
-    case 1:
+    case Post_Key:

```

Comment:
Constants are usually defined as all uppercase, i.e. `POST_KEY`.  You already did this for `CURL_ERROR`, i.e.